keps/sig-cli/993-kustomize-generators-transformers/README.md

```diff
@@ -404,6 +404,7 @@ NA - Client side only
 
 ## Implementation History
 
+- Alpha available in Kustomize (standalone binary, not kubectl kustomize) v2.1.0+ behind the `--enable-alpha-plugins` flag. https://kubectl.docs.kubernetes.io/blog/2019/06/18/v2.1.0/#generator-and-transformer-plugins
```

Comment:
@monopole can you confirm what the exact state is? I thought we had disabled plugins in kubectl entirely, but that's not the case. It seems like we [removed](https://github.com/kubernetes-sigs/kustomize/pull/3686/files) `--enable-star` and `--enable-exec` specifically, but not the more general `--enable-alpha-plugins` gate. I manually checked that container-based transformers work. Was that the intent? I'd like to leave a better note about this state here.